Lombiq.Tests.UI/Extensions/NavigationUITestContextExtensions.cs

```diff
@@ -17,6 +17,18 @@ public static class NavigationUITestContextExtensions
     public static Task GoToRelativeUrlAsync(this UITestContext context, string relativeUrl, bool onlyIfNotAlreadyThere = true) =>
         context.GoToAbsoluteUrlAsync(context.GetAbsoluteUri(relativeUrl), onlyIfNotAlreadyThere);
 
+    public static Task GoToAdminRelativeUrlAsync(
+        this UITestContext context,
+        string urlWithoutAdminPrefix = null,
+        bool onlyIfNotAlreadyThere = true)
+    {
+        if (string.IsNullOrEmpty(urlWithoutAdminPrefix)) return context.GoToDashboardAsync();
+
+        return context.GoToAbsoluteUrlAsync(
+            context.GetAbsoluteUri($"{context.AdminUrlPrefix + urlWithoutAdminPrefix}"),
```

Comment:
Can you explain in your own words how you understand this suggestion? Why did I tell it? What may be that'd be better on this line when using `UriBuilder`?

Reply:
My though was that  `$"{context.AdminUrlPrefix + urlWithoutAdminPrefix}"` isn't really a nice way of connecting 2 pieces of uri. I haven't really ever used UriBuilder before but I checked it out and I have a wild guess that apart from my usage of UriBuilder supplies the needed Uri it is still not the best as I still connecting the strings the same way. But I couldn't find a proper other way as both AdminPrefix + RelativeUrl should be in the Path property.

Reply:
Yes, exactly. The point is not to simply string concat `context.AdminUrlPrefix` and `urlWithoutAdminPrefix`. However, now I see that `UriBuilder` indeed doesn't have something like `Path.Combine()`, which I remembered it has, sorry about that. However, you can achieve the same concatenation with something like `new Uri(context.AdminUrlPrefix, urlWithoutAdminPrefix)`.

Reply:
There isn't a ctor overload for `Uri()` that takes 2 string aswell. I added Flurl library that has Url.Combine and solved it with that. It's up to you now.

Reply:
There's one that takes a `Uri` and a string or two `Uri`s.

Reply:
Isn't this Url.Combine good? There is one with 2 Uri s that I know.

Reply:
It's good but it would be better not to add and have to maintain a dependency for a one-liner that the framework provides functionality for.

![image](https://user-images.githubusercontent.com/1976647/186487959-2f85635e-6e81-4f7e-9c3a-a9cc597aaf1f.png)

Reply:
With GetAbosuleUri I don't even need to connect Uri's because this actually does that and I can reach uri.LocalPath and the Uri itself that I need.

Reply:
@Piedone Okay non of those constructors work for us. Literally they are only able to attach a base Uri to a relative One
so new `Uri("https:\\localhost:5002", "/Admin")` will result in `https:\\localhost:5002/Admin`
BUT `new Uri("localhost:5002/Admin", "/Features")` will result in `localhost:5002/Features` because `localhost:5002/Admin` is not a baseUri

Reply:
So we either go with the flurl library Url.Combine or we just + operator them together

Reply:
I went with a string.Concat and I created a `GetAsboluteAdminUrl()` method which takes a relativeUrl as a parameter will attach it inside to "/Admin".

Reply:
I see, OK then.